apps/templates/app_page.html

```
@@ -206,14 +231,74 @@ <h2 id="app-name">
+				{% else %}
+				{% for release in app.get_releases %}
+				<div class="row">
+					<div class="span8">
+						{# Release version #}
+						<h4>
+							{% if app.platform == 'desktop'%}
 							<a href="{% url 'release_download' app.name release.version %}">
 								{{ release.version }}
 							</a>
+							{% else %}
+							<a href="{% url 'release_install' app.name release.version %}">
```

HTMLHint (1.9.2)

[error] Tag must be paired, missing: [ </a> ], start tag match failed [ <a href="{% url 'release_install' app.name release.version %}"> ] on line 299.

(tag-pair)